### ISSUE TITLE
Support for CSS on different commands

### ISSUE THREAD

**Issue by Veejay** [daviferreira/medium-editor#637]
Hi,

I'd like to know if it's possible to handle say `justifyCenter` or `justifyRight` using CSS classes instead of inline style attributes?

Right now, a click on justify right generates a 

    style="text-align: right" 

attribute on the element, I'd rather it generated something like 
    
    class="text-align-right"

which I could always customize at a later time. Is such a feature supported in medium-editor?

What I tested so far:

1) Looking at styleWithCss, whose name seems to indicate that it would help handle CSS styling (which it doesn't really)
2) Fiddling with the selected element inside of `execActionInternal`, but that breaks undo/redo.

In general, it'd be nice to be able to customize which particular classes should be added on each element in the markup generated by medium-editor.

**Comment by Veejay** [comment]
@daviferreira I might be able to solve my issue with this if you know how to directly modify a selection's classList (through some `el.classList.add('something')` for example) without breaking undo.

Right now, since modifying the element's classList is not done through a command, hitting undo simple doesn't work. I don't know the undo/redo innards well enough to know if there's a way to do work outside of execCommand and still have undo/redo. Any luck for me?

**Comment by phiggins42** [comment]
you could in theory select the node, and insertHTML back in a class-less/full version to tie into undo stack, but i think it'd made 2 hits.  I am working on an undo extension providing transactional commits to allow such synchronous dom changes to happen outside of a generated/maintained undo stack, but it is far from complete just yet.

**Comment by Veejay** [comment]
@phiggins42 Yup, gave that a try, it doesn't remove the class. Thanks for the suggestion anyway, much appreciated.
